var/spack/repos/builtin/packages/neurodamus-core/package.py

```diff
@@ -45,6 +46,10 @@ def install(self, spec, prefix):
             copy_all('resources/common/hoc', prefix.hoc)
             copy_all('resources/common/mod', prefix.mod)
 
-    def setup_environment(self, spack_env, run_env):
+        filter_file(r'UNKNOWN_CORE_VERSION', r'%s' % spec.version, prefix.hoc.join('defvar.hoc'))
+        git_commit_hash = self.fetcher[0].get_commit_hash()
```

Comment:
I think we should add small check if `fetcher` is `git fetcher`. We know that it's git package but good to have a check. (if you want, you can move fetcher related line in senate method).

Reply:
You are right. It's now done :+1: